hw_3/hw_3/ViewController.swift

```diff
@@ -0,0 +1,177 @@
+    }
+    
+    func subtraction(_ firstNumber: Int,_  secondNumber: Double) {
+        let numberSum = Double(firstNumber) - secondNumber
+        print("\(firstNumber) - \(secondNumber) = \(numberSum) ")
+    }
+    
+    func subtraction(_ firstNumber: Double,_  secondNumber: Int) {
+        let numberSum = firstNumber - Double(secondNumber)
+        print("\(firstNumber) - \(secondNumber) = \(numberSum) ")
+    }
+    
+    func subtraction(_ firstNumber: Double,_  secondNumber: Double) {
+        let numberSum = firstNumber - secondNumber
+        print("\(firstNumber) - \(secondNumber) = \(numberSum) ")
+    }
+    
+    func multiply(_ firstNumber: Int,_  secondNumber: Int) {
+        let numberSum = firstNumber * secondNumber
+        print("\(firstNumber) * \(secondNumber) = \(numberSum) ")
+    }
+    
+    func multiply(_ firstNumber: Int,_  secondNumber: Double) {
+        let numberSum = Double(firstNumber) * secondNumber
+        print("\(firstNumber) * \(secondNumber) = \(numberSum) ")
+    }
+    
+    func multiply(_ firstNumber: Double,_  secondNumber: Int) {
+        let numberSum = firstNumber * Double(secondNumber)
+        print("\(firstNumber) * \(secondNumber) = \(numberSum) ")
+    }
+    
+    func multiply(_ firstNumber: Double,_  secondNumber: Double) {
+        let numberSum = firstNumber * secondNumber
+        print("\(firstNumber) * \(secondNumber) = \(numberSum) ")
+    }
+    
+    func division(_ firstNumber: Int,_  secondNumber: Int) {
+        let numberSum = Double(firstNumber) / Double(secondNumber)
+        print("\(firstNumber) / \(secondNumber) = \(numberSum) ")
+    }
+    
+    func division(_ firstNumber: Int,_  secondNumber: Double) {
+        let numberSum = Double(firstNumber) / secondNumber
+        print("\(firstNumber) / \(secondNumber) = \(numberSum) ")
+    }
+    
+    func division(_ firstNumber: Double,_  secondNumber: Int) {
+        let numberSum = firstNumber / Double(secondNumber)
+        print("\(firstNumber) / \(secondNumber) = \(numberSum) ")
+    }
+    
+    func division(_ firstNumber: Double,_  secondNumber: Double) {
+        let numberSum = firstNumber / secondNumber
+        print("\(firstNumber) / \(secondNumber) = \(numberSum) ")
+    }
+
+    func compareStrings(firstString: String, secondString: String) {
+        if firstString > secondString {
+            print("\(firstString) > \(secondString)")
+        }
+        else if firstString < secondString {
+            print("\(firstString) < \(secondString)")
+        }
+        else {
+            print("\(firstString) = \(secondString)")
+        }
+    }
+    
+    func sumFourDigitNumberDigits(number: Int) {
+        let firstNumber = number / 1000
+        let secondNumber = (number / 100) % 10
+        let thirdNumber = (number / 10) % 10
+        let fourthNumber = number % 10
+        let digitsSum = firstNumber + secondNumber + thirdNumber + fourthNumber
+        print("Сумма \(number) = \(digitsSum)")
+        
```

Comment:
You can remove this empty line from here because it doesn't split lines of code with different logic. It's better to add an empty line between `let digitsSum` and `print`